_includes/manuals/1.22/5.1_api.md

```diff
@@ -1,3 +1,3 @@
-[API v2](api/{{page.version}}/index.html) is the default, stable and recommended version for Foreman {{page.version}}.  API v1 is also available, but future versions of Foreman will eventually deprecate and remove it.
```

Comment:
IIRC we removed v1 a while back (1.20? 1.21?) so we should make sure to update those manuals as well.

Reply:
Actually, it was 1.18, see https://projects.theforeman.org/issues/11389.